### PR TITLE
[DO NOT MERGE] Add Leftpad SDK Example

### PR DESCRIPTION
**Pull request by waprin**
Experimental support for Leftpad SDK. Let me know what you think, seems like a really useful Software-as-a-Service the community might embrace.

### CODE REVIEW

**Comment by broady** [comment]
Please add tests before merging.

File: appengine/leftpad/app.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright 2015-2016, Google, Inc.
```
Comment (broady) [inline appengine/leftpad/app.js]: 2016

File: appengine/leftpad/app.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+var express = require('express');
+var leftPadIOSDK = require('left-pad-io-sdk');
+
+var app = express();
+
+app.get('/', function(req, res) {
+    leftPadIOSDK.leftPad('dog', 10, '#').then(function(response){
+          console.log(response); // logs the response 
+          res.status(200).send('response');
```
Comment (broady) [inline appengine/leftpad/app.js]: This sends the string `'response'` rather than the response itself.

File: appengine/leftpad/app.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    })
+});
+
+// Start the server
+var server = app.listen(process.env.PORT || '8080', '0.0.0.0', function() {
+  console.log('App listening at http://%s:%s', server.address().address,
+
+    server.address().port);
```
Comment (broady) [inline appengine/leftpad/app.js]: 4 spaces for continuation indents (and remove newline above)

File: appengine/leftpad/app.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+var express = require('express');
+var leftPadIOSDK = require('left-pad-io-sdk');
+
+var app = express();
+
+app.get('/', function(req, res) {
+    leftPadIOSDK.leftPad(res.query.str, res.query.len, res.query.ch).then(function(response){
+          console.log(response); // logs the response 
+          res.status(200).send(response);
```
Comment (broady) [inline appengine/leftpad/app.js]: too many spaces here. just two spaces for regular indent, then four spaces only for continuation (i.e., wrapped lines)

**Comment by JustinBeckwith** [comment]
image

**Pull request closed without merging**